ew/cmd/juviecmd/juviecmds.py

```diff
@@ -582,6 +585,10 @@ async def talk(cmd):
             npc_obj = static_npc.active_npcs_map.get(checked_npc.enemyclass)
             await npc_obj.func_ai(keyword='talk', enemy = checked_npc, channel = cmd.message.channel, user_data=user_data)
 
+    else: #No tokens, no mentions, just the bare command
+        response = "Who do you want to **!talk** to?"
```

Comment:
I imagine I'll wind up changing this for conveyance, since you use !talk with no other tokens when talking to a vendor. Everything else looks OK

Reply:
alright! honestly i barely knew anything bout the npc stuff till i did these minor changes. thanks for the correction!